src/VisualStudio/Core/Def/Implementation/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -51,27 +44,40 @@ internal partial class StreamingFindUsagesPresenter :
             IThreadingContext threadingContext,
             VisualStudioWorkspace workspace,
             Shell.SVsServiceProvider serviceProvider,
-            ITextBufferFactoryService textBufferFactoryService,
-            ITextEditorFactoryService textEditorFactoryService,
-            IContentTypeRegistryService contentTypeRegistryService,
             ClassificationTypeMap typeMap,
             IEditorFormatMapService formatMapService,
-            IClassificationFormatMapService classificationFormatMapService,
-            IProjectionBufferFactoryService projectionBufferFactoryService,
-            IEditorOptionsFactoryService editorOptionsFactoryService)
+            IClassificationFormatMapService classificationFormatMapService)
+            : this(workspace, threadingContext, serviceProvider, typeMap, formatMapService, classificationFormatMapService)
+        {
+        }
+
+        // Test only
+        public StreamingFindUsagesPresenter(
+            Workspace workspace,
+            ExportProvider exportProvider)
+            : this(workspace,
+                  exportProvider.GetExportedValue<IThreadingContext>(),
+                  exportProvider.GetExportedValue<Shell.SVsServiceProvider>(),
+                  exportProvider.GetExportedValue<ClassificationTypeMap>(),
+                  exportProvider.GetExportedValue<IEditorFormatMapService>(),
+                  exportProvider.GetExportedValue<IClassificationFormatMapService>())
```

Comment:
➡️ Same situation as https://github.com/dotnet/roslyn/pull/30733#discussion_r229942475

Reply:
same answer as before. if one believe code style is very important, please place a system that force it on all PR/CI build rather than asking each PR to do it manually.

doesn't need to be general thing. could be Roslyn only system. but make it automated.

Reply:
> same comment as above. smart indent does not produce a 6-char indent.

Cyrus and I verified that smart indent produces a 4-char indent on his machine and a 6-char indent on mine. We have no idea why this difference occurs, but it means it's plausible that this formatting was correct at the time it was written.